package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "gpii-json-schema",
+  "version": "0.1.0",
+  "description": "Fluid components to add support for JSON Schema validation.",
+  "main": "",
+  "scripts": {
+    "test": "node tests/all-tests.js",
+    "postinstall": "grunt dedupe-infusion"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GPII/gpii-json-schema"
+  },
+  "author": "Tony Atkins <tony@raisingthefloor.org>",
+  "license": "BSD-3-Clause",
+  "dependencies": {
+    "ajv": "1.4.5",
+    "grunt": "~0.4.4",
+    "grunt-contrib-jshint": "~0.9.0",
+    "grunt-gpii": "git://github.com/GPII/grunt-gpii.git#58ae055833f437a9e91baedb9f39a22c5fdd727f",
+    "grunt-jsonlint": "1.0.4",
+    "grunt-shell": "0.6.4",
+    "infusion": "git://github.com/fluid-project/infusion.git#282f1a318718eed0b0ec060fb8b4ad254417fd7e",
+    "json-schema-ref-parser": "1.4.0"
+  },
+  "devDependencies": {
+    "gpii-express": "git://github.com/the-t-in-rtf/gpii-express.git#8730441601db239870046c796b85c0f020a302a1",
```

Comment:
It looks like this is not actually a devDependency, but actually a core-dependency, because of schemaHandler and schemaMiddleware, right?